### PR TITLE
Bug 1588291 Updates to oreg_url

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1588291

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@bmcelvee this can be moved along as it has been verified.

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -664,6 +664,8 @@ openshift_examples_modify_imagestreams=true
 |Variable |Purpose
 |`*oreg_url*`
 |Set to the alternate image location. Necessary if you are not using the default registry at `registry.access.redhat.com`.
+The default component inherits the image prefix and version from `oreg_url`.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: s/oreg_url/the oreg_url value

File: install/index.adoc [hunk 2]
```diff
@@ -314,7 +314,10 @@ You can also set the `openshift_docker_insecure_registries` variable to the IP
 address of the host. `0.0.0.0/0` is not a valid setting.
 ====
 
-The configuration of additional, insecure, and blocked `docker` registries occurs
+The default component inherits the image prefix and version from `oreg_url`.
```
Comment (kalexand-rh) [inline install/index.adoc]: s/oreg_url/the oreg_url value

File: install/index.adoc [hunk 3]
```diff
@@ -314,7 +314,10 @@ You can also set the `openshift_docker_insecure_registries` variable to the IP
...
+
+The configuration of additional, insecure, and blocked Docker registries occurs
+
```
Comment (kalexand-rh) [inline install/index.adoc]: Do you need an extra line here?

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -168,12 +150,10 @@ eventrouter will be installed. When both are 'false', eventrouter will be
 uninstalled.
 
 |`openshift_logging_eventrouter_image_prefix`
-|The prefix for the eventrouter logging image. The default is set to
-`openshift_logging_image_prefix`.
+|The prefix for the eventrouter logging image.
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: \*eventrouter* ?

File: upgrading/automated_upgrades.adoc [hunk 5]
```diff
@@ -750,15 +748,13 @@ the following required variables within the `[OSEv3:vars]` section:
...
-openshift_metrics_image_version=<tag> <2>
-openshift_metrics_hawkular_hostname=<fqdn> <3>
-openshift_metrics_cassandra_storage_type=(emptydir|pv|dynamic) <4>
+openshift_metrics_hawkular_hostname=<fqdn> <2>
+openshift_metrics_cassandra_storage_type=(emptydir|pv|dynamic) <3>
 ----
 <1> Enables the ability to upgrade the metrics deployment.
-<2> Replace `<tag>` with `{latest-int-tag}` for the latest version.
-<3> Used for the Hawkular Metrics route. Should correspond to a fully qualified
+<2> Used for the Hawkular Metrics route. Should correspond to a fully qualified
```
Comment (kalexand-rh) [inline upgrading/automated_upgrades.adoc]: s/Should correspond to/Specify

File: upgrading/automated_upgrades.adoc [hunk 6]
```diff
@@ -750,15 +748,13 @@ the following required variables within the `[OSEv3:vars]` section:
...
 domain name.
-<4> Choose a type that is consistent with the previous deployment.
+<3> Choose a type that is consistent with the previous deployment.
```
Comment (kalexand-rh) [inline upgrading/automated_upgrades.adoc]: s/a type that is consistent with/the same type as

File: install_config/aggregate_logging.adoc [hunk 7]
```diff
@@ -168,12 +150,10 @@ eventrouter will be installed. When both are 'false', eventrouter will be
...
 
 |`openshift_logging_eventrouter_image_version`
-|The image version for the logging eventrouter. The default is set to
-'openshift_logging_image_version'.
+|The image version for the logging eventrouter.
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: \*eventrouter* ?

**Comment by bmcelvee** [comment]
Thanks, @kalexand-rh! :cat:

**Pull request merged**